x-pack/test/functional/apps/maps/layer_errors.js

```diff
@@ -65,14 +64,13 @@ export default function ({ getPageObjects }) {
       });
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/36011
-    describe.skip('EMSFileSource with missing EMS id', () => {
+    describe('EMSFileSource with missing EMS id', () => {
       const MISSING_EMS_ID = 'idThatDoesNotExitForEMSFileSource';
       const LAYER_NAME = 'EMS_vector_shapes';
 
       it('should diplay error message in layer panel', async () => {
         const errorMsg = await PageObjects.maps.getLayerErrorText(LAYER_NAME);
-        expect(errorMsg).to.equal(`Unable to find EMS vector shapes for id: ${MISSING_EMS_ID}`);
+        expect(errorMsg).to.contain(MISSING_EMS_ID);
```

Comment:
I'd rather have this expect include the full text of the message.  I think the only reason it doesn't currently match the product is that the text was changed while the test was skipped.